web/react/components/channel_header.jsx

```diff
@@ -194,6 +194,17 @@ export default class ChannelHeader extends React.Component {
             );
         }
 
+        const popoverListMembers = [];
+        if (!isDirect) {
+            popoverListMembers.push(
+                <PopoverListMembers
+                    members={this.state.users}
+                    memberCount={this.state.userCount}
+                    channelId={channel.id}
+                />
```

Comment:
Thank you for your comments, I pushed the update. As for having to null the variable - React renders undefined as empty string. Still, of for coding style or reliability reasons you do recommend to set it to null, I'll make this change. Please let me know